command.go

```diff
@@ -568,7 +568,7 @@ Additional Commands:{{range $cmds}}{{if (and (eq .GroupID "") (or .IsAvailableCo
 Flags:
 {{.LocalFlags.FlagUsages | trimTrailingWhitespaces}}{{end}}{{if .HasAvailableInheritedFlags}}
 
-Global Flags:
+Flags inherited from parent commands:
```

Comment:
I'm not a fan of this change: while you're _technically_ correct (what we're calling global flags are just flags inherited from the top level command), in spirit, they are global flags.

This would be a pretty confusing change for people who've used cobra for years.

Reply:
Hmmm that's a very fair point: this is probably one of those legacy inconsistencies that has popped up as we've expanded the command / flag system. It's not really global, like you said, since it is inherited from _some_ parent.

Anyone else have opinions here.

Reply:
The point is valid but I don’t think it justifies changing a formulation that tools have been using for years. 

In fact, I wonder if many tools aren’t putting the inherited flags mostly on the root command which would make them global flags   so although this printout is not technically correct, it is in practice the right thing, and what existing tools are expecting

Reply:
It's fair to say that this change would be somewhat of an inconvenience for developers using the tool and the cost may be a bit high for this change. I'll revert the print out because of that in this PR.

I think it's good though to consider that application developers aren't the end users for cobra. The end users are the ones actually using the application and the ones that would actually read the help documentation. In our case, the "Global Flags" print has actually caused a bit of confusion.

Anyway, dealing with this slight inconsistency between markdown and help func is the point of this PR and I'll revert it back for this. How this is worded would probably a discussion to have at some other point.